src/ProjectTemplates/Microsoft.Extensions.AI.Templates/src/ChatWithCustomData/ChatWithCustomData-CSharp.Web/Components/Pages/Chat/ChatSuggestions.razor

```diff
@@ -48,7 +48,7 @@
         {
             var response = await ChatClient.GetResponseAsync<string[]>(
                 [.. ReduceMessages(messages), new(ChatRole.User, Prompt)],
-                useNativeJsonSchema: true, cancellationToken: cancellation.Token);
+                useJsonSchema: true, cancellationToken: cancellation.Token);
```

Comment:
This change depends on [Enable JSON-schema based structured output by default (dotnet/extensions#6159)](https://github.com/dotnet/extensions/pull/6159), right?

Reply:
Yeah, it does. It's tricky because when there's a breaking change, it's not a simple MSBuild property to switch between the publicly available packages and the just-built packages. Do you think it's fine that I'm adapting the template to stick with the latest internal changes?

Reply:
In this case there happens to be an easier solution: we can and should now remove the `useJsonSchema` optional parameter from this line entirely. The new default value is `true` anyway.

I don't think that will cause any drawback if this gets used with the old MEAI that defaults to false, because the old version wasn't applying strict JSON mode anyway (which was a separate problem).

Reply:
Thanks. Just removed the argument.